src/ehr_components/ui/button.tsx

```diff
@@ -0,0 +1,21 @@
+export interface Prop {
```

Comment:
added all the common UI components inside the `ui` folder. Can we move autocomplete to `ui` folder itself?

Reply:
Yes, make sense